app/products/calls/connection/connection.ts

```diff
@@ -41,14 +41,20 @@ export async function newConnection(
     channelID: string,
     closeCb: (err?: Error) => void,
     setScreenShareURL: (url: string) => void,
+    setLocalVideoURL: (url: string) => void,
+    setRemoteVideoURL: (url: string) => void,
     hasMicPermission: boolean,
+    hasCameraPermission: boolean,
```

Comment:
I think we should probably refactor this as it's getting a bit verbose (we are at 10 arguments).